### PR TITLE
Add notes for longer timeout in 3.7 deployment

### PR DESCRIPTION
**Pull request by aeng**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zanata/zanata-server/pull/841" target="_blank">ReviewNinja

File: docs/release-notes.md [hunk 1]
```diff
@@ -25,7 +25,17 @@ Example usage in html file: `<link rel="shortcut icon" href="#{assets['img/logo/
 * [PR 633](https://github.com/zanata/zanata-server/pull/633) - Use JNDI to obtain mail server from app server
     * Zanata now uses `java:jboss/mail/Default` mail session for SMTP configuration.  See "Email configuration" in [System admin guide](http://docs.zanata.org/en/latest/user-guide/system-admin/configuration/installation/index.html) for details.
 
+<h5>Deployment</h5>
 
+Deployment for this release might require longer time due to underlying database changes.
```
Comment (djansen-redhat) [inline docs/release-notes.md]: I'd say "Deployment for this release may require a longer timeout"

File: docs/release-notes.md [hunk 2]
```diff
@@ -25,7 +25,17 @@ Example usage in html file: `<link rel="shortcut icon" href="#{assets['img/logo/
 * [PR 633](https://github.com/zanata/zanata-server/pull/633) - Use JNDI to obtain mail server from app server
     * Zanata now uses `java:jboss/mail/Default` mail session for SMTP configuration.  See "Email configuration" in [System admin guide](http://docs.zanata.org/en/latest/user-guide/system-admin/configuration/installation/index.html) for details.
 
+<h5>Deployment</h5>
 
+Deployment for this release might require longer time due to underlying database changes. 
+This is depending on database size and system admin should consider increase JBoss timeout value in standalone.xml.
```
Comment (djansen-redhat) [inline docs/release-notes.md]: This is dependent on database size and the system administrator should consider increasing the JBoss timeout value in standalone.xml.

**Comment by djansen-redhat** [comment]
:+1:

**Pull request merged**